scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -76,7 +76,7 @@
     __monitor__,
     "container_check_interval",
     "Optional (defaults to 5). How often (in seconds) to check if containers have been started or stopped.",
-    convert_to=int,
+    convert_to=float,
```

Comment:
This makes it easier / faster to try to reproduce the scenario locally since we can use interval smaller than 1 second and the change is also fully backward compatible.